HGGM/Services/NotificationService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using HGGM.Models;
+using HGGM.Models.Identity;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.UI.Services;
+
+namespace HGGM.Services
+{
+    public class NotificationService : INotificationService
+    {
+        private readonly IEmailSender _emailSender;
+        private readonly UserManager<User> _userManager;
+
+        public NotificationService(IEmailSender emailSender, UserManager<User> userManager)
+        {
+            _emailSender = emailSender;
+            _userManager = userManager;
+        }
+        // Missing - how to create link
+        public async Task NotifyUsers(string subject, string message, IList<string> userIdList)
+        {
+            var notification = new Notification
+            {
+                DateNotified = DateTimeOffset.Now,
+                Message = message,
+                Subject = subject,
+                Link = ""
```

Comment:
should be provided by the caller as parameter
if the caller is controller, it can be easily created with urlhelper